CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Fix the unstarted http/2 connection cancellations. (#757)
```

Comment:
Perhaps

```suggestion
- Ensure HTTP/2 connections are closed on cancellation. (#757)
```

I think the title of this pull request is a little misleading as well :)

Reply:
Thanks! I changed it to be more descriptive.